README.md

```diff
@@ -30,12 +76,73 @@ This will pull a pre-built image from Docker Hub (or use an existing image if al
 If you'd rather build the model locally you can follow the steps in the
 [model README](https://github.com/IBM/MAX-Object-Detector/#steps).
 
+#### 2. Experimenting with the API (Optional)
+
+The API server automatically generates an interactive Swagger documentation page.
+Go to `http://localhost:5000` to load it. From there you can explore the API and also create test requests.
+
+Use the `model/predict` endpoint to load a test image and get predicted labels for the image from the API.
+The coordinates of the bounding box are returned in the `detection_box` field, and contain the array of normalized
+coordinates (ranging from 0 to 1) in the form `[ymin, xmin, ymax, xmax]`.
```

Comment:
It would be helpful to add how to translate the normalized coordinates into coordinates that one would use (e.g to draw a box)

Reply:
That feature is the point of this code pattern, I don't think explaining the details of how bits of code work in the readme is good documentation for a code pattern.

Reply:
You are referring to the MAX-Object-Detector repository readme? If you are, then yes, I agree.

Reply:
I believe we're on the same page. I have a reference to the model repo's readme for more detailed documentation. As for the having an example for how to utilize the coordinates, that example is this app itself, aka the code.